### DISCUSSION TITLE
【灌水】赛博生命打过来了？？？

### DISCUSSION THREAD

**Discussion by GanlinSama** [share121/inter-knot#1481] [其他]
上图
![123](https://github.com/user-attachments/assets/cdf44c19-ee21-49f5-a95e-e83476a1d375)
![SharedScreenshot](https://github.com/user-attachments/assets/05f9af69-6e20-4fa6-8f73-e66b57847a14)
很明显这个人的回答跟楼主的问题
  **_一点都不沾边吧！_**
建议管理员监视一下

**Comment by GanlinSama** [comment]
如果不是的话我立马道歉删帖，最近见水军见的太多了，实在被打怕了

**Comment by dongyilin0928** [comment]
确实，真心希望这里环境别被水军污染了

**Comment by dhysj** [comment]
一堆水军能打到这里也真是为难他们了

**Comment by WarrenCham** [comment]
能到这里也真不容易

**Comment by cpu644** [comment]
幽默水军觉得发这也能一条三毛

**Comment by szlx2333** [comment]
y1s1绳网里几乎全是来水的，只有这哥们不是

**Comment by GanlinSama** [comment]
好了，我回原帖看了一下，评论已经被删掉了

**Comment by Meishangzhao** [comment]
我们是绳匠它是绳斤，正常

**Comment by greenhoie** [comment]
这种情况应该是文字理解的偏差，贴主应该是开个玩笑话题，而评论区的这位应该是将贴主理解为玩游戏的时候开作弊插件自动完成委托任务，很显然他情绪很激动

**Comment by GanlinSama** [comment]
我不是已经锁帖了么？怎么还有人评论

**Comment by Dave-12138** [comment]
> 我不是已经锁帖了么？怎么还有人评论

暂时没有“识别closed并将其从推送流中屏蔽”的功能，也就是虽然你关了但还是会被刷到，等更新罢

**Comment by demolute** [comment]
建议吧主硬气点，该管就管。不要像米游社一样
